### PR TITLE
Fix #1756: Change Password Button not working properly

### LINKED ISSUE #1756: Change Password Button not working in any language other than English language mode
**Summary:**

Change Password Button not working in any language other than English language mode

**Steps to reproduce:**

Watch the video

**Expected behaviour:**

Change Password Button should work in every language

**Observed behaviour:**

Change Password Button should does not work in any language other than English

**Device and Android version:**

Any

**Screenshots:**

https://user-images.githubusercontent.com/54978105/109628179-73876380-7b68-11eb-8966-0b8334acb797.mp4

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #1756

Please Add Screenshots If there are any UI changes.

https://user-images.githubusercontent.com/54978105/109628698-0f18d400-7b69-11eb-84b8-3cb3d1375a6b.mp4

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -153,6 +153,7 @@
     <string name="no">No</string>
     <string name="found">found</string>x
 
+    <string name="key">Password</string>
```
Comment (jddeep) [inline app/src/main/res/values/strings.xml]: give it a suitable name. How about trying `password_key`

**Review by jddeep — CHANGES_REQUESTED** [review]

**Review by PatelVatsalB21 — COMMENTED** [review]
@ritish099 please check with './gradlew check' --scan' before commit.

**Comment by ritish099** [comment]
@PatelVatsalB21 I cannot figure out. what's the problem.. U can see the PR. There is no such changes.. and just cloning the development branch and running "./gradlew check" is creating the error(No change in code). Can u help me?

**Comment by PatelVatsalB21** [comment]
@ritish099 I know that, it seems annotation error, clean and rebuild your project and try force push, it might help you. Sometimes it happens like this

**Pull request closed without merging**